.gitignore

```diff
@@ -1,6 +1,7 @@
 /build/
 /dist/
 /BUILD_ENV_VARS
+/.chevah_arrays
```

Comment:
can we have this file inside `./build/chevah_build_array` or something like that ?

This looks to me like `BUILD_ENV_VARS` 
Maybe we can get rid of `BUILD_ENV_VARS` and only use this file to exchage data between processes.

Reply:
I've tried to unify the files, it's possible, but ugly and confusing. So I gave up the idea. 

The `BUILD_ENV_VARS` file is used to pass the detected platform-specific values from the child script `pythia.sh` to the parent script `build.sh`.

The new file `.chevah_arrays` is used to pass the detected per-OS commands to use during build from `build.sh` to `chevahbs` files. This is done in different scripts in different ways… And the reason for this is also different: you can't pass Bash arrays to other scripts, not even from a parent script to a child one.

There's also a reason the new `.chevah_arrays` is kept in the root of the hierarchy of files: it has to be read by `chevahbs` scripts through a relative path when executed from either `src/bla-bla` or `build/bla-bla`.

In the end, I've tried to clarify to above in https://github.com/chevah/pythia/pull/44/commits/65a3240a10563a6fefc19a579cc3d2d61b3b72f5. Let me know if there's more to be done, thanks!